prpl/chat.c

```diff
@@ -221,15 +221,17 @@ static void match_contact_cb(ChimeConnection *cxn, ChimeContact *contact, gpoint
 
 static void open_participant_im(PurpleConnection *conn, GList *row, gpointer _chat)
 {
-	ChimeConnection *cxn = PURPLE_CHIME_CXN(conn);
-	struct ccb ccb = { NULL, row->data };
-
-	chime_connection_foreach_contact(cxn, match_contact_cb, &ccb);
-	if (ccb.contact) {
-		PurpleConversation *pconv = purple_conversation_new(PURPLE_CONV_TYPE_IM,
-								    purple_connection_get_account(conn),
-								    chime_contact_get_email(ccb.contact));
-		purple_conversation_present(pconv);
+	if (row != NULL) {
```

Comment:
Nitpick: I see there are other uses of `g_return_if_fail` in the code.  Perhaps it would be a better option to avoid too much indentation?

Reply:
I don't think you want to warn about a NULL row here. You just want to do nothing since it is a valid case.

Reply:
I didn't remember about the printed warning.  How about a quick return?

``` c
if (row == NULL)
        return;
```

This way the diff is short and it doesn't clobber the blame/annotation

Reply:
I am ok with that